### DISCUSSION TITLE
Rate to high error - but i will get some Data

### DISCUSSION THREAD

**Discussion by tuvok81** [jomjol/AI-on-the-edge-device#1272] [General]
Hello Guys,
iam very new in this things.. and my english is bad! :) I have a Problem with the ESP32-CAM and your Software
i get data but i have no idea about the error

![grafik](https://user-images.githubusercontent.com/30367063/200058460-adb00dc5-cb3e-468f-b38d-e97e0592b200.png)

**Comment by ghost** [comment]
Set the previous value manually. By default a check (MaxRateValue) is enabled. So worse reading ends in error. The 42 is in my opinion the start value after setup. The previous value will also unset after a timeout (720minutes by default).

**Reply by tuvok81** [reply]
OK and what do i make in MaxRateValue for a entry?

**Reply by caco3** [reply]
just set the current value as previous Value under `Settings > Set Precious Value`

**Reply by tuvok81** [reply]
OK and what for a value do i add on this section?

**Reply by caco3** [reply]
the current raw value

**Reply by tuvok81** [reply]
ok i will show thx
